src/service.ts

```diff
@@ -0,0 +1,13 @@
+import { Client } from "./client";
+
+export class Service {
```

Comment:
Seems like this class is not very useful by itself, but is being used as a base class.
If you want to avoid users being able to do `new Service(...)`, you should mark it as `abstract`.